backend/src/apiserver/main.go

```diff
@@ -50,12 +50,13 @@ const (
 )
 
 var (
-	logLevelFlag       = flag.String("logLevel", "", "Defines the log level for the application.")
-	rpcPortFlag        = flag.String("rpcPortFlag", ":8887", "RPC Port")
-	httpPortFlag       = flag.String("httpPortFlag", ":8888", "Http Proxy Port")
-	configPath         = flag.String("config", "", "Path to JSON file containing config")
-	sampleConfigPath   = flag.String("sampleconfig", "", "Path to samples")
-	collectMetricsFlag = flag.Bool("collectMetricsFlag", true, "Whether to collect Prometheus metrics in API server.")
+	logLevelFlag         = flag.String("logLevel", "", "Defines the log level for the application.")
+	rpcPortFlag          = flag.String("rpcPortFlag", ":8887", "RPC Port")
+	httpPortFlag         = flag.String("httpPortFlag", ":8888", "Http Proxy Port")
+	configPath           = flag.String("config", "", "Path to JSON file containing config")
+	sampleConfigPath     = flag.String("sampleconfig", "", "Path to samples")
+	collectMetricsFlag   = flag.Bool("collectMetricsFlag", true, "Whether to collect Prometheus metrics in API server.")
+	useKubernetesStorage = flag.Bool("store-in-kubernetes", false, "Store and run pipeline versions in Kubernetes")
```

Comment:
This flag should likely be specific to pipelines being stored in Kubernetes as we might add others in the future.